artemis/db.py

```diff
@@ -28,6 +48,72 @@ def as_dict(self) -> Dict[str, Any]:
             assert False
 
 
+Base = declarative_base()
+engine = create_engine(
+    Config.Data.POSTGRES_CONN_STR, json_serializer=functools.partial(json.dumps, cls=JSONEncoderAdditionalTypes)
+)
+Session = sessionmaker(bind=engine)
+
+
+class TSVector(TypeDecorator):  # type: ignore
+    impl = TSVECTOR
+
+
+class ScheduledTask(Base):  # type: ignore
+    __tablename__ = "scheduled_task"
+    created_at = Column(DateTime, server_default=text("NOW()"))
```

Comment:
```suggestion
    created_at = Column(DateTime, default=datetime.datetime.now())
```

Reply:
This will evaluate now() only once, when the class is created. What I proposed will create a timestamp once a row is added. This will be converted to "DEFAULT NOW()" in SQL, so we won't need to provide the date on INSERT.